### ISSUE TITLE
Application <application name> has not been registered.

### ISSUE THREAD

**Issue by eric90066** [inspireui/mstore#88]
I've successfully complied my app using the instructions in the manual.  I am trying to connect to my wordpress woocommerce site. 

When I run the app in genymotion the above error is thrown (with the name of my app as "application name") and the following:

Hint: This error often happens when you're running the packager (local dev server) from a wrong folder. For example you have multiple apps and the packager is still running for the app you were working on before.
If this is the case, simply kill the old packager instance (e.g. close the packager terminal window) and start the packager in the correct app folder (e.g. cd into app folder and run 'npm start').

This error can also happen due to a require() error during initialization or failure to call AppRegistry.registerComponent.

runApplication
    AppRegistry.js:169:6
__callFunction
    MessageQueue.js:260:47
<unknown>
    MessageQueue.js:101:26
__guard
    MessageQueue.js:228:6
callFunctionReturnFlushedQueue
    MessageQueue.js:100:17

As instructed in the manual, I have:

android\/app\/src\/main\/java\/com\/piksal\/mstore\/MainActivity.java
app/src/main/java/com/piksal/mstore/MainApplication.java

in folder: 
/android/app/src/main/java/com/yourpackage/name

Clearly there is an issue with the configuration here.  Unsure what to do next.  Please advise. 

BTW - I am using Android Studio with mstore-pro

**Comment by eric90066** [comment]
For anyone else with this issue,  the solution for me was to add the name of my store to /mstore-pro/index.android.js where it says beostore.  

Be careful.  Only delete the word Beostore.  DO NOT delete the single quotes around it ('bestore').  For whatever reason, when I deleted the word and quotes and retyped the single quotes the app would not recognize the quotes correctly. 

In addition, I also edited and renamed beostore.imi in /android/ to nameofmystore.ini (where nameofmystore is the name of your store or project).  I did this to the id (the second line in the file) as well. 

Sadly, none of this is mentioned in the manual.
